packages/app/src/containers/ThemeSymbolsInspector.tsx

```diff
@@ -29,11 +30,14 @@ export default memo(function ThemeSymbolsInspector() {
 
   if (selectedSymbols.length === 0) return null;
 
+  const showSymbolsInspector = selectedSymbols.length === 1;
```

Comment:
Do we need this? Within `SymbolInspector` we already make sure we only have 1

Reply:
I think we should keep it for now since the inspector it's currently only working on the first symbol. Unless it should show even when selecting multiple symbols

Reply:
I don't have any way of showing conflicting values, in the fields I am showing 🤔 Like when you we show "Multiples" in the input when you are selecting multiple things but they don't share the same value for that field

Reply:
Oh yeah, good point, we should keep it